proposals/0279-multiple-trailing-closures.md

```diff
@@ -245,6 +247,43 @@ when(2 < 3,
 )
 ```
 
+Note that "unlabeled" closures are also allowed when "empty" label placeholder `_:` is used to delimit each closure e.g.
+
+```swift
+func foo(_ fn1: () -> Void, _ fn2: () -> Void) {}
+
+foo {
+  _: { ... }
+  _: { ... }
+}
+```
+
+An attempt to use multiple trailing closures syntax with a single unlabeled closure argument are going to be treated as if
+it used a single trailing closure syntax instead, but if label is specified it's going to be preserved e.g.
+
+```swift
+func foo(fn: () -> Void) {
+}
+
+foo {
+  _: { ... }
+}
+
+foo {
+  fn: { ... }
+}
+```
+
+Would be transformed into:
+
+```swift
+foo { ... }
```

Comment:
Minor nit: can you write this as 
```swift
foo({ ... })
```
to eliminate the trailing closure entirely?

Reply:
Yeah, one sec.

Reply:
Done!